### PR TITLE
Add MousePosition control

### PR DESCRIPTION
**Pull request by oterral**

### CODE REVIEW

**Comment by edellucien** [comment]
Super minor comment, apart that very nice! :)

File: src/components/MousePosition/MousePosition.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+   */
+  projections: PropTypes.arrayOf(
+    PropTypes.shape({
+      /**
+       * The label to display in the select box.
+       */
+      label: PropTypes.string.isRequired,
+
+      /**
+       * The value used to create the options´s projection of the MousePosition control. See [doc](https://openlayers.org/en/latest/apidoc/module-ol_control_MousePosition.html)
```
[the remark this answers is not in the document]
Comment (oterral) [reply src/components/MousePosition/MousePosition.js]: Done

File: src/components/MousePosition/MousePosition.test.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      });
+    });
+
+    test('using only one projection', () => {
+      expectSnapshot({
+        projections: [{ label: 'foo', value: 'foo' }],
+      });
+    });
+
+    test('using multiple projectionds.', () => {
```
[the remark this answers is not in the document]
Comment (oterral) [reply src/components/MousePosition/MousePosition.test.js]: Done

**Comment by mhaertwig** [comment]
Nothing to complain. Thanks!

**Pull request merged**